canonicalwebteam/blog/ub_wordpress.py

```diff
@@ -0,0 +1,52 @@
+from canonicalwebteam.blog import Wordpress
+from canonicalwebteam.blog.helpers import transform_article
+
+
+class UbWordpress(Wordpress):
+    def __init__(
+        self,
+        session,
+        api_url="https://admin.insights.ubuntu.com/wp-json/wp/v2",
+        transform_links=True,
+    ):
+        super().__init__(session, api_url)
+
+        self.transform_links = transform_links
+
+    def get_articles(
+        self,
+        tags=None,
+        tags_exclude=None,
+        exclude=None,
+        categories=None,
+        sticky=None,
+        before=None,
+        after=None,
+        author=None,
+        groups=None,
+        per_page=12,
+        page=1,
+    ):
+        articles, metadata = super().get_articles(
+            tags,
+            tags_exclude,
+            exclude,
+            categories,
+            sticky,
+            before,
+            after,
+            author,
+            groups,
+            per_page,
+            page,
+        )
+
+        return (
+            [transform_article(a, self.transform_links) for a in articles],
```

Comment:
`transform_article` is really the whole point of this class - it's basically the only thing that differentiates it from `Wordpress` right now. Could we move that helper function inside this class instead? `def _transform_article`?

Reply:
I moved `transform_article` inside the class. I moved the rest of the functions from the `helpers` file that were used only by `transform_article` too. 

I deleted the functions from the `helpers` file that were unused. 

In the end I was left with two functions which I moved to the `BlogViews` class, which allows me to get rid of the `helpers` file completely.